bin/sdl_make_tracklooper

```diff
@@ -18,21 +18,22 @@ usage()
   echo "      sh $(basename $0) OPTIONSTRINGS ..."
   echo
   echo "Options:"
-  echo "  -h    Help                    (Display this message)"
-  echo "  -c    cache                   (Make library with cache enabled)"
-  echo "  -s    show log                (Full compilation script to stdout)"
-  echo "  -m    make clean binaries     (Make clean binaries before remake. e.g. when header files changed in SDL/*.h)"
-  echo "  -d    cut value ntuple        (With extra variables in a debug ntuple file)"
-  echo "  -p    primitive object ntuple (With extra variables related to primitive objects)"
-  echo "  -3    do T3T3 extensions      (-e turned on if not specified)"
-  echo "  -N    neural networks         (Toggle LST neural networks)"
-  echo "  -C    cpu backend             (Run code on CPU, serially)"
+  echo "  -h    Help                      (Display this message)"
+  echo "  -c    cache                     (Make library with cache enabled)"
+  echo "  -s    show log                  (Full compilation script to stdout)"
+  echo "  -m    make clean binaries       (Make clean binaries before remake. e.g. when header files changed in SDL/*.h)"
+  echo "  -d    cut value ntuple          (With extra variables in a debug ntuple file)"
+  echo "  -p    primitive object ntuple   (With extra variables related to primitive objects)"
+  echo "  -3    do T3T3 extensions        (-e turned on if not specified)"
+  echo "  -N    neural networks           (Toggle LST neural networks)"
+  echo "  -C    cpu backend               (Run code on CPU, serially)"
+  echo "  -2    no pLS duplicate cleaning (Toggle LST neural networks)"
```

Comment:
Think the stuff in parentheses was copied over incorrectly for the pLS flag.

Reply:
Oops, right, thanks for spotting it. Fixed in [4203e70](https://github.com/SegmentLinking/TrackLooper/pull/335/commits/4203e70e7a0e39daf72f5069c1793c8daf19ee66).